Taskfile.yml

```diff
@@ -72,3 +74,29 @@ tasks:
         diff
         <(cd '{{.OUTPUT_DIR}}' && tar cf - {{.CHECKSUM_TAR_BASE_ARGS}} . | md5sum)
         "{{.CHECKSUM_FILE}}"
+
+  serve:
+    deps:
+      - "http-server"
+      - "docs"
+    cmds:
+      - "npm exec --prefix '{{.NODE_DEPS_DIR}}' http-server '{{.DOCS_OUTPUT_DIR}}' -c-1"
```

Comment:
There isn't enough proper official documentation about how `--prefix` works and I didn't know it was an `npm` option rather than an `npm i`/`npm exec` option. 

I looked it up on the Internet and this is what I found:
https://github.com/npm/cli/issues/1368#issuecomment-1241076700
> I think we're gonna close this as a wontfix as --prefix is not the best flag to be promoting. We do eventually want to refactor or remove it but for now we should at least not be promoting it.

Let's hope they don't remove it.

Reply:
Nice find.